src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import duke.commands.Command;
+
+/**
+ * This class represents the Duke chatbot. It is the main class from which the chatbot is run.
+ */
+public class Duke {
+    private TaskList tasks;
+    private final Storage storage;
+    private final Ui ui;
+
+    /**
+     * Constructs a Duke object.
+     * 
+     * @param filePath Path of file where list of tasks is stored.
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        this.storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (DukeException e) {
+            ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+    
+    public static void main(String[] args) {
```

Comment:
Perhaps you can add in a javadocs comment here since all methods other than getter and setters need to have it?